code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -151,7 +151,10 @@ var/const/MAX_ACTIVE_TIME = 400
 			M.Paralyse(MAX_IMPREGNATION_TIME/6) //something like 25 ticks = 20 seconds with the default settings
 	else if(iscorgi(M))

```

Comment:
Yeah, it's weird that corgis are the only non-carbons that are huggable. You could make a PR for that, I suppose...